src/components/RichTextEditor/utils/editorConfig.ts

```diff
@@ -169,3 +179,28 @@ export const getEditorConfig = () => {
         },
     );
 };
+
+export const EditorConfigAnnotation = () => {
```

Comment:
What is the reason for that?
Couldn't we just take the editorConfig like before and manage with the editorActions which plugins we want to use and which not?

Reply:
The reason for that is, if I add to normal config I will have mentions always in every RTE even if I dont want to use it, cant control it over action.